frontend/src/components/chat/MessageBubble.css

```
@@ -0,0 +1,40 @@
+:root {
+  --blue: #83bffe;
+  --gray: #cccccc;
+}
+
+.chat-message-bubble {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 1rem;
+  font-family: "Tenor Sans", sans-serif;
+}
+
+.chat-message-bubble__elements {
+  max-width: 75%;
+}
+
+.chat-message-bubble__elements--theirs {
+  align-self: flex-start;
+}
+
+.chat-message-bubble__elements--mine {
+  align-self: flex-end;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.chat-message-bubble__elements--theirs .chat-message-bubble__content {
+  background: var(--gray);
+}
+
+.chat-message-bubble__elements--mine .chat-message-bubble__content {
+  background: var(--blue);
```

```suggestion
  background: var(--light-green);
```

I think we could use light green for the user's own message bubble in order to have the color scheme match the rest of the site.